CharacterCreation/07_Items.txt

```diff
@@ -2,164 +2,178 @@
 separate instructions.
 
-Repair kit: $100
-For normal damage (50% or less) roll percents under 50. On pass, repair 30 + 
+Repair kit ($100):
+    For normal damage (50% or less) roll percents under 50. On pass, repair 30 + 
 percent rolled. For severe wounds (50% or more), roll percents under 50. On 
 pass, repair 30.
-If you have Skill in Repair, see the skill sheet for separate instructions.
 
-Nanite Pack: $100
-Restores 2d10 + 20 nanites.
+    If you have Skill in Repair, see the skill sheet for separate instructions.
+
+Nanite Pack ($100):
+    Restores 2d10 + 20 nanites.
 
-Security Spike: $100
-+20 to a hack check on a door with a physical access panel. DC 30 to place. 
+Security Spike ($100):
+    +20 to a hack check on a door with a physical access panel. DC 30 to place. 
 Once hacked the door can be controlled remotely.
 
-Medic Needle Pack: $60
-Contains 10 needles suitable for use with several malpractice proceedures.
+Medic Needle Pack ($60):
+    Contains 10 needles suitable for use with several malpractice proceedures.
 Not recomended for medicianal use. 
 
-Single Use door brute-forcer $120:
-Hacks through the encryption on a door (up to 75 DC) and opens it.
+Door Brute-Forcer ($120):
+    Hacks through the encryption on a door (up to 75 DC) and opens it. Single 
+use device.
 
 Zip-Cuffs ($25):
-pack of 5 nilon fiber woven standard zip-tie cuffs. Strength DC 70 or INT 80 
+    Pack of 5 nilon fiber woven standard zip-tie cuffs. Strength DC 70 or INT 80 
 to break.
 
 HD Zip-Cuffs ($300):
-Pack of 5 carbon-fiber zipe-tie cuffs. Strength DC 95 or INT 105 to Break.
+    Pack of 5 carbon-fiber zipe-tie cuffs. Strength DC 95 or INT 105 to Break.
 
 ==============================
 Explosives
 ==============================
-Breaching charges ($500):
-Will remove a door from your path. Make sure no one is within 2m of the door
+Breaching charges ($160):
+    Will remove a door from your path. Make sure no one is within 2m of the door
 in any direction or they take damage as if hit by an ECSC fragmentation 
 grenade.
 
-Wall Charge ($840):
-Reduces a 1.5x1.5m section of wall to rubble, allowing entry. Clear areas 2m 
-surrounding the charge or take 70 explosive damage.
-Wall thicknesses:
-Wood - Up to 12in.
-concrete - up to 10in
-Reinforced Concrete - up to 8 in
-Steel plate - up to 2 in
+Wall Charge ($120):
+    Reduces a 1.5x1.5m section of wall to rubble, allowing entry. Clear areas 2m 
+surrounding the charge or take 70 explosive damage. Thicknesses of several 
+materials one wall charge can breach through are listed below.
 
-Det Cord ($230 per 2m of cord):
-Used to remove doors from hinges, act as ignition sources, enhance another 
-explosive, etc. Use 2m per door to be removed. .5m to wrap around a door 
-handle. 1m to wrap around a grenade. 4m to create a wall breaching pad (
-Explosives 50 DC)
+Wood:       Up to 12in.
+Concrete:   up to 10in
+Reinforced Concrete: up to 8 in
+Steel plate: up to 2 in
```

Comment:
Blocker: I am not certain what this will be read as. If we delete the preceding blank line will the parser interpret the DC lines as part of the preceding paragraph? (that would be good).

We could also turn DC lines into an unordered list.

Reply:
This is set up as a stat block. We could instead do a table? If we wrap it into the paragraph, it will render them all on the same line, not the prettiest.

Reply:
I didn't see stat block when I was looking at the base styling doc and thus forgot they exist. mb.

Reply:
sounds like my bad ;)